src/main/java/com/resource/TwitterWorks.java

```diff
@@ -0,0 +1,60 @@
+package com.resource;
+
+import org.eclipse.jetty.util.StringUtil;
+import twitter4j.TwitterException;
+import javax.ws.rs.*;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+
+@Produces(MediaType.APPLICATION_JSON)
+@Path("/api/1.0/twitter")
+public class TwitterWorks {
+    @POST
+    @Path("/postTweet")
+    @Consumes(MediaType.APPLICATION_JSON)
+    public Response sendTweet(TweetPostRequest tweetPostRequest){
+        String tweet=tweetPostRequest.getMessage();
+        System.out.println(tweet);
+        if(StringUtil.isEmpty(tweet)) {
+            return Response.status(400,"Please Enter a valid tweet").build();
+        }
+        else {
+            try {
+                SendTweet.sendTweets(tweet);
+            }
+            catch (TwitterException e)
+            {
+                return Response.status(500,"Request Was Not Completed").build();
+            }
+            return Response.status(200,"Tweeted Successfully").build();
+        }
+    }
+    @GET
+    @Path("/getTimeline")
+    public String[] timeLine()
+    {
+           int size=RetrieveTweets.latestTweet().length;
+            String ar[]=new String[size];
```

Comment:
Same review, 
Please do not use String array
We have Collections in JAva that do all the hard work for us
https://www.javatpoint.com/collections-in-java

Reply:
yes i did by ArrayList object.